### PR TITLE
required input

### PR DESCRIPTION
**Pull request by kandrelczyk**
Hi,
Quick feature, marking field as required. Same as https://react.fluentui.dev/?path=/docs/components-field--docs#required

Let me know if anything should be changed.

### CODE REVIEW

File: thaw/src/field/field.css [hunk 1]
```diff
@@ -12,6 +12,11 @@
     color: var(--colorNeutralForeground1);
 }
 
+.thaw-field__label--required {
+    padding: 0 var(--spacingHorizontalXS);
+    color: var(--colorPaletteRedForeground1);
```
Comment (luoxiaozero) [inline thaw/src/field/field.css:17]: Modify to the following style to ensure consistency with FluentUI.

```css
padding-left: var(--spacingHorizontalXS);
color: var(--colorPaletteRedForeground3);
```

File: thaw/src/field/field.rs [hunk 2]
```diff
@@ -16,6 +16,9 @@ pub fn Field(
     /// The orientation of the label relative to the field component.
     #[prop(optional, into)]
     orientation: MaybeSignal<FieldOrientation>,
+    //Is this input field required
```
Comment (luoxiaozero) [inline thaw/src/field/field.rs:19]: Use document comments(`///`), not regular comments(`//`).

**Review by luoxiaozero — CHANGES_REQUESTED** [review]

File: thaw/src/field/field.css (hunk 1 above)
Comment (kandrelczyk) [reply thaw/src/field/field.css]: done

File: thaw/src/field/field.rs (hunk 2 above)
Comment (kandrelczyk) [reply thaw/src/field/field.rs]: done

**Comment by luoxiaozero** [comment]
Thanks!

**Pull request merged**